### PR TITLE
translate ukrainian/english (latin unchanged)

### PR DESCRIPTION
**Pull request by KurochkaR**
add translate to taxi service app templates

### CODE REVIEW

**Review by SergeyKutsko — COMMENTED** [review]
@WARGDOOM please QA

File: taxi_service/templates/about.html [hunk 1]
```diff
@@ -79,14 +78,14 @@
       <div class="row">
         <div class="col-md-6">
           <div class="img-box">
-            <img src="/static/app/images/about-img.png" alt="">
+            <img src="{% static 'app/images/about-img.png' %}" alt="">
```
Comment (WARGDOOM) [inline taxi_service/templates/about.html:81]: @KurochkaR dont works now on local, please recheck yourself

Comment (KurochkaR) [reply taxi_service/templates/about.html:81]: change settings file try again

**Comment by WARGDOOM** [comment]
how to switch languages?

**Comment by KurochkaR** [comment]
Add en to path localhost/en/

**Comment by SergeyKutsko** [comment]
@KurochkaR could you make it based on browser settings?

**Comment by KurochkaR** [comment]
@ ok add button to navbar?

**Comment by KurochkaR** [comment]
need replace static files from taxi_service/static/app to static/app for normal download pictures

**Comment by WARGDOOM** [comment]
how to switch languages ​​in the browser? should it automatically switch under different conditions?

**Comment by KurochkaR** [comment]
site get language from settings your browser

**Comment by KurochkaR** [comment]
image
1st language will work if it's not en or uk it will be uk

**Comment by KurochkaR** [comment]
https://user-images.githubusercontent.com/120567985/222120793-ef879be1-4da9-41ef-ba6a-c43d8dfa1207.mp4

**Comment by WARGDOOM** [comment]
It does not work as in the video.. only when google translation is enabled.
There is no single option. How to include it?
немає такої функції

**Comment by KurochkaR** [comment]
this option(green) just show that UI for your browser will be on that language it's turn on with check mark in language menu

**Comment by KurochkaR** [comment]
image

**Comment by KurochkaR** [comment]
But it doesn't matter, websites will show content in your preferred languages, when possible that's mean if your 1st language will be english and site have this option(en language) it'll be english, if there no language what you want (ex.french is 1st language) it'll be in ukrainian

**Comment by WARGDOOM** [comment]
Does not work!! Try installing the following version of chromе
Screenshot 2023-03-02 at 16 58 40
 and try again to match the versions. Do you have any options why it's not working?

**Comment by KurochkaR** [comment]
Is language not change at all and always ukrainian?

**Comment by KurochkaR** [comment]
Try it i think must work

**Pull request merged**